.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
 
 <!-- Please add direct links where your modifications can be seen in the documentation -->
 
-* https://deploy-preview-{your pr number}--boosted.netlify.app/
+* https://deploy-preview-{your_pr_number}--boosted.netlify.app/
 
 ### Checklist
 
 <!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 
+#### Contribution
+
 - [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+
+#### Design
```

Comment:
Yeah might be difficult for folks on their own

Reply:
Please try to suggest some things then.

Only came out with something like:

```
#### A11y

- [ ] My development follows a11y good practices
```

Reply:
It's a bit difficult to make a checklist for accessibility because there are too many possible options depending if the PR is about forms, images, links, tables,...

Reply:
@julien-deramond your proposition is good (with maybe accessibility instead of a11y ?), and you could also add to run Axe on developer work station...

Reply:
Tried something in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1706/commits/45946d4eec062d04fccdd55c5f02cc5c7d6eb02d. I'm resolving this conversation, don't hesitate to re-comment if the wording is not good or the text is not complete enough.